charts/posthog/templates/temporal-py-worker-hpa.yaml

```
@@ -0,0 +1,25 @@
+{{- if and .Values.temporalPyWorker.enabled .Values.temporalPyWorker.hpa.enabled -}}
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ template "posthog.fullname" . }}-worker
+  labels: {{- include "_snippet-metadata-labels-common" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    kind: Deployment
+    apiVersion: apps/v1
+    name: {{ template "posthog.fullname" . }}-temporalPyWorker
```

```suggestion
    name: {{ template "posthog.fullname" . }}-temporal-py-worker
```

if what I read above is correct